package_format/example_main_page/parameters.json

```diff
@@ -0,0 +1,38 @@
+{
+	name: "example-main-page"
+	image:
+	cmd: [
+		"--cas", {type: "parameter", name: "cas"},
+	]
+	requires: {
+		"cas": {
+			type: "http_service",
+			api: "cas"
+			inject: {type: "cmd", index: 3}
+		}
+		"services": {
+			type: "list"
+			of: {
+				type: "dict"
+				content: {
+					"public_path": {
+						description: "public path"
+						type: "string" // path?
+					}
+					"service": {
+						description: "internal service"
+						type: "http_service"
+						api: {type: "any"}
+					}
+				}
+			}
+		}
+	]
+	provides: [
+		{
+			type: "http_service"
+			api: "main_page"
```

Comment:
To jest typ interfejsu. Jest on po to, żeby zapewnić dodatkowe sprawdzanie poprawności.

Np API typu CAS będzie miało endpointy /login, /validate, ... które działają w określony sposób.